aim/sdk/repo.py

```diff
@@ -109,6 +110,17 @@ def __init__(self, path: str, *, read_only: bool = None, init: bool = False):
             self.root_path = path
         self.path = os.path.join(self.root_path, get_aim_repo_name())
 
+        self.container_pool: Dict[ContainerConfig, Container] = WeakValueDictionary()
+        self.persistent_pool: Dict[ContainerConfig, Container] = dict()
+        self.container_view_pool: Dict[ContainerConfig, Container] = WeakValueDictionary()
+
+        self._run_props_cache_hint = None
+        self._encryption_key = None
+        self.structured_db = None
+
+        self._lock_path = os.path.join(self.path, '.repo_lock')
```

Comment:
no need to create lock for `Repo` with remote tracking.

Reply:
good point